### ISSUE TITLE
Update 2.0.0 and 2.0.1 Connection Error to Serial Device

### ISSUE THREAD

**Issue by infoleptic** [hassio-addons/addon-zwave-js-ui#584]
# Problem/Motivation

Updated to v2.0.0 and the USB serial device fails to initialize
Updated to v2.0.1 with same results.

## Expected behavior

USB serial device connects and Aeotec Gen5 device is available.

## Actual behavior

USB serial device does not connect, no z-wave controller or associated devices available.
zwave js ui logs:
                                   · SoftReset (0x08)
                                    · GetProtocolVersion (0x09)
                                    · SetRFReceiveMode (0x10)
                                    · UNKNOWN_FUNC_SET_SLEEP_MODE (0x11)
                                    · FUNC_ID_ZW_SEND_NODE_INFORMATION (0x12)
                                    · SendData (0x13)
                                    · SendDataMulticast (0x14)
                                    · GetControllerVersion (0x15)
                                    · SendDataAbort (0x16)
                                    · FUNC_ID_ZW_R_F_POWER_LEVEL_SET (0x17)
                                    · UNKNOWN_FUNC_SEND_DATA_META (0x18)
                                    · FUNC_ID_ZW_GET_RANDOM (0x1c)
                                    · GetControllerId (0x20)
                                    · UNKNOWN_FUNC_MEMORY_GET_BYTE (0x21)
                                    · UNKNOWN_FUNC_MEMORY_PUT_BYTE (0x22)
                                    · UNKNOWN_FUNC_MEMORY_GET_BUFFER (0x23)
                                    · UNKNOWN_FUNC_MEMORY_PUT_BUFFER (0x24)
                                    · EnterBootloader (0x27)
                                    · GetNVMId (0x29)
                                    · ExtNVMReadLongBuffer (0x2a)
                                    · ExtNVMWriteLongBuffer (0x2b)
                                    · ExtNVMReadLongByte (0x2c)
                                    · ExtExtWriteLongByte (0x2d)
                                    · GetNodeProtocolInfo (0x41)
                                    · HardReset (0x42)
                                    · FUNC_ID_ZW_REPLICATION_COMMAND_COMPLETE (0x44)
                                    · FUNC_ID_ZW_REPLICATION_SEND_DATA (0x45)
                                    · AssignReturnRoute (0x46)
                                    · DeleteReturnRoute (0x47)
                                    · RequestNodeNeighborUpdate (0x48)
                                    · ApplicationUpdateRequest (0x49)
                                    · AddNodeToNetwork (0x4a)
                                    · RemoveNodeFromNetwork (0x4b)
                                    · FUNC_ID_ZW_CREATE_NEW_PRIMARY (0x4c)
                                    · FUNC_ID_ZW_CONTROLLER_CHANGE (0x4d)
                                    · FUNC_ID_ZW_SET_LEARN_MODE (0x50)
                                    · AssignSUCReturnRoute (0x51)
                                    · FUNC_ID_ZW_REQUEST_NETWORK_UPDATE (0x53)
                                    · SetSUCNodeId (0x54)
                                    · DeleteSUCReturnRoute (0x55)
                                    · GetSUCNodeId (0x56)
                                    · UNKNOWN_FUNC_SEND_SUC_ID (0x57)
                                    · FUNC_ID_ZW_EXPLORE_REQUEST_INCLUSION (0x5e)
                                    · RequestNodeInfo (0x60)
                                    · RemoveFailedNode (0x61)
                                    · IsFailedNode (0x62)
                                    · ReplaceFailedNode (0x63)
                                    · UNKNOWN_FUNC_UNKNOWN_0x66 (0x66)
                                    · UNKNOWN_FUNC_UNKNOWN_0x67 (0x67)
                                    · GetRoutingInfo (0x80)
                                    · UNKNOWN_FUNC_LOCK_ROUTE_RESPONSE (0x90)
                                    · GetPriorityRoute (0x92)
                                    · SetPriorityRoute (0x93)
                                    · UNKNOWN_FUNC_UNKNOWN_0x98 (0x98)
                                    · UNKNOWN_FUNC_UNKNOWN_0xB4 (0xb4)
                                    · UNKNOWN_FUNC_WATCH_DOG_ENABLE (0xb6)
                                    · UNKNOWN_FUNC_WATCH_DOG_DISABLE (0xb7)
                                    · UNKNOWN_FUNC_WATCH_DOG_KICK (0xb8)
                                    · UNKNOWN_FUNC_UNKNOWN_0xB9 (0xb9)
                                    · UNKNOWN_FUNC_RF_POWERLEVEL_GET (0xba)
                                    · UNKNOWN_FUNC_GET_LIBRARY_TYPE (0xbd)
                                    · UNKNOWN_FUNC_SEND_TEST_FRAME (0xbe)
                                    · UNKNOWN_FUNC_GET_PROTOCOL_STATUS (0xbf)
                                    · UNKNOWN_FUNC_UNKNOWN_0xD2 (0xd2)
                                    · UNKNOWN_FUNC_UNKNOWN_0xD3 (0xd3)
                                    · UNKNOWN_FUNC_UNKNOWN_0xD4 (0xd4)
                                    · undefined (0xee)
                                    · UNKNOWN_FUNC_UNKNOWN_0xEF (0xef)
2023-09-28T02:40:15.205Z CNTRLR   querying version info...
2023-09-28T02:40:15.230Z CNTRLR   received version info:
                                    controller type: Static Controller
                                    library version: Z-Wave 3.95
2023-09-28T02:40:15.230Z CNTRLR   querying protocol version info...
2023-09-28T02:40:15.243Z CNTRLR   received protocol version info:
                                    protocol type:             Z-Wave
                                    protocol version:          3.95.0
2023-09-28T02:40:15.243Z CNTRLR   querying controller IDs...
2023-09-28T02:40:15.257Z CNTRLR   received controller IDs:
                                    home ID:     0xfb5288ff
                                    own node ID: 1
2023-09-28T02:40:15.266Z CNTRLR   Performing soft reset...
2023-09-28T02:40:15.273Z CNTRLR   Waiting for the controller to reconnect...
2023-09-28T02:40:16.773Z CNTRLR   Waiting for the Serial API to start...
2023-09-27 21:40:19.454 INFO APP: GET /health/zwave 301 1.431 ms - 191
2023-09-28T02:40:21.773Z CNTRLR   Did not receive notification that Serial API has started, checking if it respo
                                  nds...
2023-09-28T02:40:21.776Z CNTRLR   Failed to execute controller command after 1/3 attempts. Scheduling next try i
                                  n 100 ms.
2023-09-28T02:40:21.876Z CNTRLR   Failed to execute controller command after 2/3 attempts. Scheduling next try i
                                  n 1100 ms.
2023-09-28T02:40:22.978Z CNTRLR   Serial API did not respond, trying again in 2 seconds...
2023-09-28T02:40:24.980Z CNTRLR   Failed to execute controller command after 1/3 attempts. Scheduling next try i
                                  n 100 ms.
2023-09-28T02:40:25.081Z CNTRLR   Failed to execute controller command after 2/3 attempts. Scheduling next try i
                                  n 1100 ms.
2023-09-28T02:40:26.183Z CNTRLR   Serial API did not respond, trying again in 5 seconds...
2023-09-28T02:40:31.186Z CNTRLR   Failed to execute controller command after 1/3 attempts. Scheduling next try i
                                  n 100 ms.
2023-09-28T02:40:31.289Z CNTRLR   Failed to execute controller command after 2/3 attempts. Scheduling next try i
                                  n 1100 ms.

dmesg from host shows USB device disconnecting/reconnecting repeatedly.
dmesg output:
Sep 28 02:30:15 homeassistant kernel: usb 2-2.2: New USB device found, idVendor=0658, idProduct=0200, bcdDevice= 0.00
Sep 28 02:30:15 homeassistant kernel: usb 2-2.2: New USB device strings: Mfr=0, Product=0, SerialNumber=0
Sep 28 02:30:15 homeassistant kernel: cdc_acm 2-2.2:1.0: ttyACM0: USB ACM device
Sep 28 02:30:17 homeassistant systemd[1]: run-docker-runtime\x2drunc-moby-76e69cc47aec8dbe0e742c1a5e7e0a56827ed4ee51b8de808e68f95777b365df-runc.BN456n.mount: Deactivated successfully.
Sep 28 02:30:47 homeassistant systemd[1]: run-docker-runtime\x2drunc-moby-b66a9ecfba085c08252782c7b5606b2d15759e3ca41fd2a21eef677a0a25814f-runc.niLshw.mount: Deactivated successfully.
Sep 28 02:31:04 homeassistant kernel: usb 2-2.2: USB disconnect, device number 7
Sep 28 02:31:05 homeassistant kernel: usb 2-2.2: new full-speed USB device number 8 using uhci_hcd
Sep 28 02:31:05 homeassistant kernel: usb 2-2.2: New USB device found, idVendor=0658, idProduct=0200, bcdDevice= 0.00
Sep 28 02:31:05 homeassistant kernel: usb 2-2.2: New USB device strings: Mfr=0, Product=0, SerialNumber=0
Sep 28 02:31:05 homeassistant kernel: cdc_acm 2-2.2:1.0: ttyACM0: USB ACM device
Sep 28 02:31:15 homeassistant systemd[1]: run-docker-runtime\x2drunc-moby-c4372839a8b6452c2e780f1276792b81cb7b99925d2b9703c8e193acac0493f2-runc.xTTkTS.mount: Deactivated successfully.
Sep 28 02:31:17 homeassistant systemd[1]: run-docker-runtime\x2drunc-moby-b66a9ecfba085c08252782c7b5606b2d15759e3ca41fd2a21eef677a0a25814f-runc.K55Wln.mount: Deactivated successfully.
Sep 28 02:31:17 homeassistant systemd[1]: run-docker-runtime\x2drunc-moby-76e69cc47aec8dbe0e742c1a5e7e0a56827ed4ee51b8de808e68f95777b365df-runc.wtNbDi.mount: Deactivated successfully.
Sep 28 02:31:45 homeassistant systemd[1]: run-docker-runtime\x2drunc-moby-c4372839a8b6452c2e780f1276792b81cb7b99925d2b9703c8e193acac0493f2-runc.oUmNAb.mount: Deactivated successfully.
Sep 28 02:31:47 homeassistant systemd[1]: run-docker-runtime\x2drunc-moby-b66a9ecfba085c08252782c7b5606b2d15759e3ca41fd2a21eef677a0a25814f-runc.4YXzkw.mount: Deactivated successfully.
Sep 28 02:32:00 homeassistant kernel: usb 2-2.2: USB disconnect, device number 8
Sep 28 02:32:00 homeassistant kernel: usb 2-2.2: new full-speed USB device number 9 using uhci_hcd
Sep 28 02:32:01 homeassistant kernel: usb 2-2.2: New USB device found, idVendor=0658, idProduct=0200, bcdDevice= 0.00
Sep 28 02:32:01 homeassistant kernel: usb 2-2.2: New USB device strings: Mfr=0, Product=0, SerialNumber=0
Sep 28 02:32:01 homeassistant kernel: cdc_acm 2-2.2:1.0: ttyACM0: USB ACM device
Sep 28 02:32:15 homeassistant systemd[1]: run-docker-runtime\x2drunc-moby-c4372839a8b6452c2e780f1276792b81cb7b99925d2b9703c8e193acac0493f2-runc.5CE3IV.mount: Deactivated successfully.
Sep 28 02:32:17 homeassistant systemd[1]: run-docker-runtime\x2drunc-moby-b66a9ecfba085c08252782c7b5606b2d15759e3ca41fd2a21eef677a0a25814f-runc.InxAhD.mount: Deactivated successfully.
Sep 28 02:32:17 homeassistant systemd[1]: run-docker-runtime\x2drunc-moby-76e69cc47aec8dbe0e742c1a5e7e0a56827ed4ee51b8de808e68f95777b365df-runc.wV0LgB.mount: Deactivated successfully.
Sep 28 02:32:45 homeassistant systemd[1]: run-docker-runtime\x2drunc-moby-c4372839a8b6452c2e780f1276792b81cb7b99925d2b9703c8e193acac0493f2-runc.Mw4ZRW.mount: Deactivated successfully.
Sep 28 02:32:47 homeassistant systemd[1]: run-docker-runtime\x2drunc-moby-b66a9ecfba085c08252782c7b5606b2d15759e3ca41fd2a21eef677a0a25814f-runc.ol7lTu.mount: Deactivated successfully.
Sep 28 02:32:48 homeassistant systemd[1]: run-docker-runtime\x2drunc-moby-76e69cc47aec8dbe0e742c1a5e7e0a56827ed4ee51b8de808e68f95777b365df-runc.gmKeMP.mount: Deactivated successfully.
Sep 28 02:33:03 homeassistant kernel: usb 2-2.2: USB disconnect, device number 9
Sep 28 02:33:03 homeassistant kernel: usb 2-2.2: new full-speed USB device number 10 using uhci_hcd
Sep 28 02:33:04 homeassistant kernel: usb 2-2.2: New USB device found, idVendor=0658, idProduct=0200, bcdDevice= 0.00
Sep 28 02:33:04 homeassistant kernel: usb 2-2.2: New USB device strings: Mfr=0, Product=0, SerialNumber=0
Sep 28 02:33:04 homeassistant kernel: cdc_acm 2-2.2:1.0: ttyACM0: USB ACM device
Sep 28 02:33:17 homeassistant systemd[1]: run-docker-runtime\x2drunc-moby-b66a9ecfba085c08252782c7b5606b2d15759e3ca41fd2a21eef677a0a25814f-runc.aY6N6d.mount: Deactivated successfully.
Sep 28 02:33:18 homeassistant systemd[1]: run-docker-runtime\x2drunc-moby-76e69cc47aec8dbe0e742c1a5e7e0a56827ed4ee51b8de808e68f95777b365df-runc.6PYj6k.mount: Deactivated successfully.
Sep 28 02:33:45 homeassistant systemd[1]: run-docker-runtime\x2drunc-moby-c4372839a8b6452c2e780f1276792b81cb7b99925d2b9703c8e193acac0493f2-runc.0HOnP3.mount: Deactivated successfully.
Sep 28 02:33:48 homeassistant systemd[1]: run-docker-runtime\x2drunc-moby-b66a9ecfba085c08252782c7b5606b2d15759e3ca41fd2a21eef677a0a25814f-runc.JvOJlZ.mount: Deactivated successfully.
Sep 28 02:33:48 homeassistant systemd[1]: run-docker-runtime\x2drunc-moby-76e69cc47aec8dbe0e742c1a5e7e0a56827ed4ee51b8de808e68f95777b365df-runc.uuj7Ov.mount: Deactivated successfully.
Sep 28 02:34:05 homeassistant kernel: usb 2-2.2: USB disconnect, device number 10
Sep 28 02:34:05 homeassistant kernel: usb 2-2.2: new full-speed USB device number 11 using uhci_hcd
Sep 28 02:34:06 homeassistant kernel: usb 2-2.2: New USB device found, idVendor=0658, idProduct=0200, bcdDevice= 0.00
Sep 28 02:34:06 homeassistant kernel: usb 2-2.2: New USB device strings: Mfr=0, Product=0, SerialNumber=0
Sep 28 02:34:06 homeassistant kernel: cdc_acm 2-2.2:1.0: ttyACM0: USB ACM device
Sep 28 02:34:15 homeassistant systemd[1]: run-docker-runtime\x2drunc-moby-c4372839a8b6452c2e780f1276792b81cb7b99925d2b9703c8e193acac0493f2-runc.KId84f.mount: Deactivated successfully.
Sep 28 02:34:18 homeassistant systemd[1]: run-docker-runtime\x2drunc-moby-b66a9ecfba085c08252782c7b5606b2d15759e3ca41fd2a21eef677a0a25814f-runc.Jlnwit.mount: Deactivated successfully.
Sep 28 02:34:18 homeassistant systemd[1]: run-docker-runtime\x2drunc-moby-76e69cc47aec8dbe0e742c1a5e7e0a56827ed4ee51b8de808e68f95777b365df-runc.0ZAirs.mount: Deactivated successfully.
Sep 28 02:34:46 homeassistant systemd[1]: run-docker-runtime\x2drunc-moby-c4372839a8b6452c2e780f1276792b81cb7b99925d2b9703c8e193acac0493f2-runc.Z70dj1.mount: Deactivated successfully.
Sep 28 02:34:48 homeassistant systemd[1]: run-docker-runtime\x2drunc-moby-b66a9ecfba085c08252782c7b5606b2d15759e3ca41fd2a21eef677a0a25814f-runc.1S6M62.mount: Deactivated successfully.
Sep 28 02:35:07 homeassistant kernel: usb 2-2.2: USB disconnect, device number 11
Sep 28 02:35:08 homeassistant kernel: usb 2-2.2: new full-speed USB device number 12 using uhci_hcd
Sep 28 02:35:08 homeassistant kernel: usb 2-2.2: New USB device found, idVendor=0658, idProduct=0200, bcdDevice= 0.00
Sep 28 02:35:08 homeassistant kernel: usb 2-2.2: New USB device strings: Mfr=0, Product=0, SerialNumber=0
Sep 28 02:35:08 homeassistant kernel: cdc_acm 2-2.2:1.0: ttyACM0: USB ACM device
Sep 28 02:35:18 homeassistant systemd[1]: run-docker-runtime\x2drunc-moby-b66a9ecfba085c08252782c7b5606b2d15759e3ca41fd2a21eef677a0a25814f-runc.B1mibG.mount: Deactivated successfully.
Sep 28 02:35:18 homeassistant systemd[1]: run-docker-runtime\x2drunc-moby-76e69cc47aec8dbe0e742c1a5e7e0a56827ed4ee51b8de808e68f95777b365df-runc.9K6TfG.mount: Deactivated successfully.
Sep 28 02:35:46 homeassistant systemd[1]: run-docker-runtime\x2drunc-moby-c4372839a8b6452c2e780f1276792b81cb7b99925d2b9703c8e193acac0493f2-runc.UcjnvQ.mount: Deactivated successfully.
Sep 28 02:35:48 homeassistant systemd[1]: run-docker-runtime\x2drunc-moby-b66a9ecfba085c08252782c7b5606b2d15759e3ca41fd2a21eef677a0a25814f-runc.FxaMpi.mount: Deactivated successfully.
Sep 28 02:35:49 homeassistant systemd[1]: run-docker-runtime\x2drunc-moby-76e69cc47aec8dbe0e742c1a5e7e0a56827ed4ee51b8de808e68f95777b365df-runc.VcUc9c.mount: Deactivated successfully.
Sep 28 02:36:09 homeassistant kernel: usb 2-2.2: USB disconnect, device number 12
Sep 28 02:36:09 homeassistant kernel: hrtimer: interrupt took 33295542 ns
Sep 28 02:36:09 homeassistant kernel: usb 2-2.2: new full-speed USB device number 13 using uhci_hcd
Sep 28 02:36:10 homeassistant kernel: usb 2-2.2: New USB device found, idVendor=0658, idProduct=0200, bcdDevice= 0.00
Sep 28 02:36:10 homeassistant kernel: usb 2-2.2: New USB device strings: Mfr=0, Product=0, SerialNumber=0
Sep 28 02:36:10 homeassistant kernel: cdc_acm 2-2.2:1.0: ttyACM0: USB ACM device
Sep 28 02:36:15 homeassistant kernel: kauditd_printk_skb: 38 callbacks suppressed
Sep 28 02:36:15 homeassistant kernel: audit: type=1334 audit(1695868575.045:381): prog-id=82 op=LOAD
Sep 28 02:36:15 homeassistant systemd[1]: Started Journal Gateway Service.
Sep 28 02:36:15 homeassistant systemd-timesyncd[441]: Network configuration changed, trying to establish connection.
Sep 28 02:36:15 homeassistant systemd-timesyncd[441]: Contacted time server 162.159.200.123:123 (time.cloudflare.com).
Sep 28 02:36:15 homeassistant systemd-journal-gatewayd[8739]: microhttpd: MHD_OPTION_EXTERNAL_LOGGER is not the first option specified for the daemon. Some messages may be printed by the standard MHD logger.
Sep 28 02:36:16 homeassistant systemd[1]: run-docker-runtime\x2drunc-moby-c4372839a8b6452c2e780f1276792b81cb7b99925d2b9703c8e193acac0493f2-runc.c0bVoX.mount: Deactivated successfully.

## Steps to reproduce

VMware HassOS OVA on ESXi v7U3d
USB passthrough of Aeotec z-stick gen5
Homeassistant Core 2023 9.3
Upgrade zwave js ui from v1.16.0 to 2.0.0 or 2.0.1
Upgrading VM hardware version from v10 to v19 did not resolve the issue.

The same VM restored from backup with zwave js ui v1.16.0 works fine.

## Proposed changes

> (If you have a proposed change, workaround or fix,
> describe the rationale behind it)

**Comment by tarvip** [comment]
I have same issue, also, reverted back to 1.16.0.
Also Aeotec z-stick gen5 USB passthrough mode on Synology NAS. I noticed that USB device is getting disconnected from VM after connecting it.

**Comment by mattzuba** [comment]
Disabling Soft Reset in the ZWave settings fixed this for me.

**Comment by mirkin-pixel** [comment]
Same issues, error: recovering unresponsive controller.. driver failed

**Comment by albatorsk** [comment]
I can confirm that disabling **Soft Reset** under **Z-Wave JS -> Settings (cogwheel) -> Z-Wave** resolves the issue for me as well on an Aeotec Z-Stick Gen5.

**Comment by nsleigh** [comment]
> I can confirm that disabling **Soft Reset** under **Z-Wave JS -> Settings (cogwheel) -> Z-Wave** resolves the issue for me as well on an Aeotec Z-Stick Gen5.

Same issue for me and this fixed it. Thanks.

**Comment by nechry** [comment]
Also with an Aeotec Z‐Stick Gen5+, need to disable Soft Reset to start network.
i'm also under VMware HassOS OVA (ESXi 7.0 Update 3)

**Comment by AlCalzone** [comment]
500 series in a VM requiring soft-reset to be disabled rings a bell: https://github.com/zwave-js/node-zwave-js/pull/6256

`node-zwave-js` previously tried to auto-detect whether a controller would come back after soft-resetting, but this feature had false negatives, so it was removed in `v12`.

**Comment by nechry** [comment]
> @nechry which firmware version is your stick on?
firmware Version is 1.02

"productLabel": "ZW090",
"productDescription": "Z‐Stick Gen5 USB Controller",
"manufacturer": "AEON Labs",
"firmwareVersion": "1.2",
"sdkVersion": "6.81.6",
"protocolVersion": 3,

**Comment by AlCalzone** [comment]
Ok, so not firmware-version related then :(

AFAIK the issue is the following: Some 500 series controllers reconnect to USB when being restarted (soft-reset). Some VMs require specific configuration to be able to handle this, otherwise the connection to the controller gets lost.

The proper solution would be to configure the VM correctly (whatever correct is), the easy solution is disabling soft-reset.

**Comment by mattster98** [comment]
> the easy solution is disabling soft-reset

Sorry, I'm having a hard time figuring out how to do this.  Can you point me in the right direction?

**Comment by nsleigh** [comment]
For me it was in the Z-Wave JS UI, then settings, the zwave section.

**Comment by mattster98** [comment]
Apologies, I ended up in the wrong bug asking that question.  I'm using the zwave_js Add-on and the associated integration.  I don't think I'm using the z-wave JS UI.  I'll hop back to the other bug (which I hope is the right one to ask)

**Comment by infoleptic** [comment]
Disabling 'Soft Reset' from the Z-Wave JS UI settings page stopped the constant USB disconnect/reconnect messages in dmesg.  The Z-Stick then began to function correctly. 

It's odd that the upgrade to v2.0.x triggered this issue, but if that's the full resolution, that's fine.

**Comment by krische** [comment]
Disabling "Soft Reset" also worked for me, but only after I rebooted my VM running Home Assistant.

**Comment by AlCalzone** [comment]
Documentation of the issue and a list of solutions can be found here:
https://github.com/zwave-js/node-zwave-js/issues/6341

**Comment by vinmar1** [comment]
I'm on a Synology DS920+ and it actually removed my Aeotec stick from the VM configuration. I added it back in and then restored to my last VM snapshot (I do nightly's). I went into the Z-wave JS UI and disabled the Soft Reset as suggested here then performed the upgrade again with no issues.

**Comment by FirstRulez** [comment]
I have an Aeotec Z-Stick Gen5 connected to my HP MicroServer running unRaid with USB pass through to the VM running Home Assistant.
![image](https://github.com/hassio-addons/addon-zwave-js-ui/assets/32237385/8a78395a-d4e3-4ad1-81b8-3c6737005b68)
Since the update to 2.0.1 I am seeing the error "Failed to open the serial port: Error: No such file or directory"
![image](https://github.com/hassio-addons/addon-zwave-js-ui/assets/32237385/ba71d3ac-5688-40e1-9eba-91afea51216a)
I have tried disabling the Soft Reset and restarted the add-on but no success.

Any pointers would be much appreciated. I am currently reverting my addon to v1.16.0 to check that works and hopefully have working lights again.

**Comment by AlCalzone** [comment]
@FirstRulez Please make a [driver log](https://zwave-js.github.io/zwave-js-ui/#/troubleshooting/generating-logs?id=driver-logs), loglevel `debug` and attach it here as a file (drag & drop into the text field).

**Comment by FirstRulez** [comment]
I will have to do that Sunday (involves me upgrading and breaking the lights control again which the boss will not approve until then!)

**Comment by deviantintegral** [comment]
I ran into the same issue - also running in a VM, unfortunately on a system that doesn’t support PCI passthrough so reconnecting USB devices is tricky.

Could this be added to the change log? It’s an easy enough fix but I would never have guessed I could fix it in the zwavejs UI.

**Comment by firefighter4700** [comment]
Hi. The "soft reset" did not work for me. I am also out of luck with the backups because of worktravel i did not get a chance to save my earliest backup before they were overwritten. I am defininatly changing my backup polisy to several backups before they get auto deleted and no more autoupdate for me.

Anyway; does anyone got a basic instruction, for a newbee, on how to rollback the version on zwave js ui to 1.16.0 or is it possibe to just uninstall the 2.0.1 and install 1.16.0 directly?

**Comment by jbhorner** [comment]
I have also experienced this issue. I am running HA Proxmox 8.0.4 in a VM (versus a container), am using USB pass-through, and have tried to resolve by turning "soft reset" off. I am using a Zooz ZST 10--not the 700 series, but the older one; I believe this is akin to using the Aeotec Gen5 but am not sure.

As I always do before I update any significant updates, I took a snapshot before updating to 2.0.1. After trying everything suggested to resolve the issue, I ultimately reverted to the snapshot which brought back my working Z-Wave network. I have a Zooz 800 that I will replace the existing Zooz with, but not soon. I see it as a rainy-day project where I am stuck inside my home.

I'm happy to provide any other information as needed. This has not been a good six weeks for Z-Wave on Home Assistant. Prior to this issue there was another one where devices were not communicating properly with the Z-Wave coordinator. It also required a revert back to a snapshot until that one was resolved. Fortunately, I only have 12 Z-Wave devices; the bulk of my devices are Zigbee and I'm adding Matter devices when needed.

**Comment by AlCalzone** [comment]
> I'm happy to provide any other information as needed

Please make a [driver log](https://zwave-js.github.io/zwave-js-ui/#/troubleshooting/generating-logs?id=driver-logs), loglevel `debug` and attach it here as a file (drag & drop into the text field).

**Comment by FirstRulez** [comment]
> @FirstRulez Please make a [driver log](https://zwave-js.github.io/zwave-js-ui/#/troubleshooting/generating-logs?id=driver-logs), loglevel `debug` and attach it here as a file (drag & drop into the text field).

Apologies for the slow responses. Preparing to go on holiday so it's a fine balance between trying to work this out and not breaking things before going away.

Yesterday I spent some time looking at this, and started with trying to update the firmware on my Zstick because it's below the recommended v1.2
```
  "firmwareVersion": "1.1",
  "sdkVersion": "6.51.10",
```
I failed completely to get the update to work, and took the stick back to re-connect from my Windows 11 PC to the server.
This caused me to identify another issue which is that the unRAID server is not automatically re-connecting the USB to the VM after it disconnects. I had to power down the VM, reconfigure the USB pass-through settings then boot up again.
This could well be linked to my original error, but I think I need a few hours to look at it in more detail.
For now I have v1.16 installed, working and will leave it that way until after my holiday when I will start again.

**Comment by AlCalzone** [comment]
> This caused me to identify another issue which is that the unRAID server is not automatically re-connecting the USB to the VM after it disconnects. I had to power down the VM, reconfigure the USB pass-through settings then boot up again.

@FirstRulez sounds like you needed to do that after disabling soft-reset in the first place 👍️

**Comment by jbhorner** [comment]
> > I'm happy to provide any other information as needed
> 
> Please make a [driver log](https://zwave-js.github.io/zwave-js-ui/#/troubleshooting/generating-logs?id=driver-logs), loglevel `debug` and attach it here as a file (drag & drop into the text field).

[zwave-js-ui-store.zip](https://github.com/hassio-addons/addon-zwave-js-ui/files/12805645/zwave-js-ui-store.zip)

Here you go...I think I did it correctly.

**Comment by deviantintegral** [comment]
And mine!

[zwavejs_2023-10-04.log](https://github.com/hassio-addons/addon-zwave-js-ui/files/12811123/zwavejs_2023-10-04.log)

**Comment by vectorxy** [comment]
> > This caused me to identify another issue which is that the unRAID server is not automatically re-connecting the USB to the VM after it disconnects. I had to power down the VM, reconfigure the USB pass-through settings then boot up again.
> 
> @FirstRulez sounds like you needed to do that after disabling soft-reset in the first place 👍️

I made the mistake of updating to HA 2023.10.0 yesterday thinking that the older zwave js ui 1.16.0 would continue to work. Problem is it won't and complains the older version will not work with the new HA update. I tried the newest version of zwave js but it's still is broken on my unraid vm setup. I've tried killing the soft reset option, unplugging the dongle and even tried reconfiguring the USB passthrough on unraid. None fixed my issue and I can't role back HA to the older version and it's stuck on 2023.10.0 . Pretty much my zwave network is dead in the water. I'm debating on decoupling my zwave js ui service from the vm and moving it to separate docker instance on unraid. Not really sure the best way to migrate the entities meta data over.

**Comment by AlCalzone** [comment]
@jbhorner that was helpful - this is actually a bug in most (all?) 500 series controllers that was uncovered by changes in Z-Wave JS v12. I finished a workaround earlier and added another PR that enables it for your controller: https://github.com/zwave-js/node-zwave-js/pull/6372

**Comment by firefighter4700** [comment]
Have anyone tried the 2.0.2 version? Are we back to normal with this? Had to uninstall my js ui addon and move all my z wave divices to the js addon. Really want to swich back again..

**Comment by AlCalzone** [comment]
I'm still ironing out a few kinks, release should go out in the next hour - then Z-Wave JS UI needs to pick it up.

**Comment by jbhorner** [comment]
Quick update on this one. The issue I was having with my Zooz 10 (500 series adapter) appears to be resolved with the [v9.1.0] based on the pop-up I received when I opened the UI, or 2.1.0 if I look at the JS UI add-on version. (The latter being what was shown as an upgrade from 1.16 this morning [2023.10.06 US PT].) I've been running it for a few hours with no visible issues. Prior to this, the 2.0.1 and 2.0.2 versions would immediately stop functioning correctly in my environment.

Many thanks to all who participated in helping to overcome the 500 series issues.

**Comment by FirstRulez** [comment]
> > This caused me to identify another issue which is that the unRAID server is not automatically re-connecting the USB to the VM after it disconnects. I had to power down the VM, reconfigure the USB pass-through settings then boot up again.
> 
> @FirstRulez sounds like you needed to do that after disabling soft-reset in the first place 👍️

I am now back from holidays, and have done the following:
1. tried to firmware update my zstick which failed miserably. I think I may have such an old one it does not support the firmware updated.
2. disabled soft-reset
3. updated Home Assistant to 2023.10.1
4. updated z-wave JS UI addon to 2.1.1
5. updated z-wave configuration to use /dev/serial/by-id/usb-0658_0200-if00 instead of /dev/ttyACM0
6. rebooted the VM which is running Home Assistant

Everything seems to be working again which is good.

Would you recommend I look at replacing the zstick with a 700 series to avoid future problems?

**Comment by AlCalzone** [comment]
> Would you recommend I look at replacing the zstick with a 700 series to avoid future problems?

If you are in the US: Maybe, as long as you get one that's between firmware 7.17.2 and 7.18.x. The effective range may be lower than the 500 series.
If you are in the EU: no

**Comment by Wiigian** [comment]
Any news on this?
I installed the latest availble update on the evening october 10th, which must have been z-wave JS UI addon 2.1.0 afaik. I ran into a lot of issues with my 500 series stick being disconnected, so I had to roll back.
2.1.0 is using node-zwave-js 12.0.3 which had https://github.com/zwave-js/node-zwave-js/pull/6372 implemented.
In 2.1.1, there are some additional tweaks done (https://github.com/zwave-js/node-zwave-js/pull/6378), so this might fixed now?

**Comment by AlCalzone** [comment]
All the issues I know about are fixed or worked around since 12.0.4. There are some reports that it doesn't work, but none include driver logs, so I'm not sure if it's just a user error.

**Comment by Wiigian** [comment]
I can confirm that the issue is still present for me in 12.0.4. Within hours after updating, my z-wave stick became unresponsive several times, and disconnected. I have now to reverted to 11.14.2, which has proven to be stable in my setup.
I am running HA supervised in a Proxmox VM.

**Comment by AlCalzone** [comment]
There have been a couple more fixes, we're now at `12.2.0`, released everywhere AFAIK:
https://zwave-js.github.io/which-version/

**Comment by xekil** [comment]
same error :(

**Comment by Wiigian** [comment]
> same error :(

Which version did you try?

**Comment by Wiigian** [comment]
3.0.2 is out, using node-zwave-js 12.3.0, which should sort this issue. I'm trying it now. Thanks @AlCalzone !

**Comment by Wiigian** [comment]
Unfortunately, it only lasted a bit less than three hours before I got the first warning about the controller being unresponsive. I'll try to disable the soft reset functionality

**Comment by k8gg** [comment]
Late to the party, and I am trying to understand this issue - is this related to a certain hardware, with 500 series chips?? Or a pointer would be appreciated regarding where I can lookup such information / such discussion...?
